ansible/inventories/devnet-7/group_vars/besu.yaml

```diff
@@ -23,15 +23,15 @@ besu_container_env:
 besu_container_volumes:
   - "{{ besu_datadir }}:/data"
   - "{{ besu_auth_jwt_path }}:/execution-auth.jwt:ro"
-  - "{{ eth_testnet_config_dir }}:/network-config:ro"
+  - "{{ eth_testnet_config_dir }}:/data/network-config:ro"
```

Comment:
is there some reason to make this change? its just a path in the container

Reply:
also its not great practice to do this:
`"{{ besu_datadir }}:/data"` and then `"{{ eth_testnet_config_dir }}:/data/network-config:ro"`, cuz then you have `eth_testnet_config_dir` showing up in `besu_datadir` on the host.